src/SPC/util/GlobalEnvManager.php

```diff
@@ -55,10 +53,73 @@ public static function init(?BuilderBase $builder = null): void
                 self::putenv("SPC_LINUX_DEFAULT_CXX={$arch}-linux-musl-g++");
                 self::putenv("SPC_LINUX_DEFAULT_AR={$arch}-linux-musl-ar");
                 self::putenv("SPC_LINUX_DEFAULT_LD={$arch}-linux-musl-ld");
```

Comment:
I would like to move away from this, but probably better to do when I finish zig support and 16.0 releases.